[2022-05-12 12:00:13,139] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_B scheduled__2022-05-11T00:00:00+00:00 [queued]>
[2022-05-12 12:00:13,176] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_B scheduled__2022-05-11T00:00:00+00:00 [queued]>
[2022-05-12 12:00:13,181] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-05-12 12:00:13,190] {taskinstance.py:1250} INFO - Starting attempt 1 of 1
[2022-05-12 12:00:13,192] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-05-12 12:00:13,227] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): training_model_B> on 2022-05-11 00:00:00+00:00
[2022-05-12 12:00:13,235] {standard_task_runner.py:52} INFO - Started process 16030 to run task
[2022-05-12 12:00:13,245] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_B', 'scheduled__2022-05-11T00:00:00+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpbon8vxvz', '--error-file', '/tmp/tmpyo6ph18r']
[2022-05-12 12:00:13,249] {standard_task_runner.py:80} INFO - Job 40: Subtask training_model_B
[2022-05-12 12:00:13,370] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.training_model_B scheduled__2022-05-11T00:00:00+00:00 [running]> on host c0c6f6bf1c08
[2022-05-12 12:00:13,482] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_B
AIRFLOW_CTX_EXECUTION_DATE=2022-05-11T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-11T00:00:00+00:00
[2022-05-12 12:00:13,490] {python.py:175} INFO - Done. Returned value was: 9
[2022-05-12 12:00:13,543] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_B, execution_date=20220511T000000, start_date=20220512T120013, end_date=20220512T120013
[2022-05-12 12:00:13,635] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-12 12:00:13,767] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
